packages/web/components/TechnologyForm/Costs/index.js

```diff
@@ -52,6 +52,24 @@ const Costs = ({ form, data }) => {
 
 	return (
 		<Wrapper>
+			<Row>
+				<Cell>
+					<CurrencyInputField
+						form={form}
+						label="Qual o preço de venda dessa tecnologia?"
+						name="technologyCosts.price"
+						placeholder="R$"
+						validation={{ required: true }}
+					/>
+				</Cell>
+				<Cell>
+					<SwitchField
+						form={form}
+						name="technologyCosts.is_seller"
+						label="Essa tecnologia é vendida por você?"
+					/>
```

Comment:
**If it doesn't take you too much time**, it would be cool to align this component vertically on the line...it's not aligned so far...

Reply:
@luizeboli I decided to stack these fields. They looked a bit odd side by side, taking up too much space.

![image](https://user-images.githubusercontent.com/10481028/99155258-176e0180-2695-11eb-991c-b137d2d019fb.png)